### PR TITLE
[test] add cucumber tests for pending feedback controller

### PR DESCRIPTION
**Pull request by kimhoang73**

### CODE REVIEW

File: features/pending_feedback_controller.feature [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    And the field "engaged" should be filled with "Mostly agree"
+    And the field "engaged_text" should be filled with "They were engaged"
+    And the field "communication" should be filled with "Mostly agree"
+    And the field "communication_text" should be filled with "They communicated well"
+    And the field "understanding" should be filled with "Mostly agree"
+    And the field "understanding_text" should be filled with "They understood well"
+    And the field "effectiveness" should be filled with "Mostly agree"
+    And the field "effectiveness_text" should be filled with "They were effective"
+    And the field "satisfied" should be filled with "Mostly agree"
+    And the field "satisfied_text" should be filled with "I am satisfied"
```
Comment (Jldeatrick) [inline features/pending_feedback_controller.feature]: Might want to change these to a more concise version, but other than that, looks good!

File: features/pending_feedback_controller.feature [hunk 2]
```diff
@@ -0,0 +1,66 @@
+Feature: get iteration feedback from customers who haven't submitted feedback yet
+	As a customer
```
Comment (kyriakipapadimitriou) [inline features/pending_feedback_controller.feature]: You could make this a bit more precise like you did for the iterations' feedback test and specify the user is logged in.

**Review by Jldeatrick — APPROVED** [review]
Very extensive testing! Only thing I would change would be refactoring the "field" filled out steps

**Review by kyriakipapadimitriou — APPROVED** [review]
Everything looks great. Great tests!

**Review by kyriakipapadimitriou — APPROVED** [review]
Everything looks great. Good work!

**Pull request merged**